src/app/shared/file-editing.ts

```diff
@@ -17,6 +17,8 @@
 import { Directive } from '@angular/core';
 import { AlertService } from './alert/state/alert.service';
 import { Files } from './files';
+import { WorkflowVersion } from './swagger';
+import { Tag } from './swagger';
```

Comment:
Also minor/optional, but you should just have import statement for WorkflowVersion and Tag. IDE should do this for you automatically: `import { Tag, WorkflowVersion }from './swagger';`